katsdptelstate/rdb_writer.py

```diff
@@ -167,19 +168,21 @@ def _encode_item(self, key, key_dump):
                         help='Comma-separated list of Redis keys to write to RDB. [default=all]')
     parser.add_argument('--redis', type=endpoint_parser(DEFAULT_PORT),
                         default=endpoint_parser(DEFAULT_PORT)('localhost'),
-                        help='host[:port] of the Redis instance to connect to. [default=localhost:{}]'
-                             .format(DEFAULT_PORT))
+                        help='host[:port] of the Redis instance to connect to. '
+                             '[default=localhost:{}]'.format(DEFAULT_PORT))
```

Comment:
I think just `[default=%(default)s]` would work - Endpoint should have a `__str__`.

Reply:
Done (not tested).